exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,24 @@
+function greetings(mentor) {
+
 var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+var mentor2 = "Chizim";
```

Comment:
You haven't used any of the `mentor1`, `mentor2`, etc. variables in your solution.

Reply:
I fixed it. :+1: